### PR TITLE
[plugin.audio.eco99music] v0.0.4

### PR DESCRIPTION
**Pull request by noamsto**
### Description
Eco99music audio Add-on

v0.0.4 (09-10-2019):

Switched to use 'xmltodict' instead of BeautifulSoup because of wrong
XML files parsing.
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.audio.eco99music/addon.xml [hunk 1]
```diff
@@ -1,25 +1,26 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.audio.eco99music" name="eco99music" version="0.0.3" provider-name="Nir Elbaz">
+<addon id="plugin.audio.eco99music" name="eco99music" version="0.0.4" provider-name="Nir Elbaz &amp; Noam Stolero">
```
Comment (Rechi) [inline plugin.audio.eco99music/addon.xml]: separate providers with a comma (`,`)

File: plugin.audio.eco99music/addon.xml [hunk 2]
```diff
@@ -1,25 +1,26 @@
...
-        <import addon="script.module.beautifulsoup4" version="4.5.3"/>
         <import addon="script.module.requests" version="2.12.4"/>
+        <import addon="script.module.xmltodict"/>
     </requires>
     <extension point="xbmc.python.pluginsource" library="addon.py">
         <provides>audio</provides>
     </extension>
     <extension point="xbmc.addon.metadata">
-        <summary lang="he_IL">אלפי פלייליסטים וסטים מוזיקליים, ערוכים ומוכנים להאזנה, ומותאמים לכל מצב, בכל מקום ובכל שעה</summary>
+        <summary lang="he">אלפי פלייליסטים וסטים מוזיקליים, ערוכים ומוכנים להאזנה, ומותאמים לכל מצב, בכל מקום ובכל שעה</summary>
```
Comment (Rechi) [inline plugin.audio.eco99music/addon.xml]: `he_IL`

File: plugin.audio.eco99music/addon.xml [hunk 3]
```diff
@@ -1,25 +1,26 @@
...
         <summary lang="en_GB">Thousands of premium music playlists for any mood, activity and time!</summary>
-        <description lang="he_IL">
+        <description lang="he">
```
Comment (Rechi) [inline plugin.audio.eco99music/addon.xml]: `he_IL`

Comment (noamsto) [reply plugin.audio.eco99music/addon.xml]: Fixed

File: plugin.audio.eco99music/addon.xml (hunk 2 above)
Comment (noamsto) [reply plugin.audio.eco99music/addon.xml]: Fixed

File: plugin.audio.eco99music/addon.xml (hunk 1 above)
Comment (noamsto) [reply plugin.audio.eco99music/addon.xml]: Fixed

File: plugin.audio.eco99music/addon.py [hunk 4]
```diff
@@ -8,100 +8,129 @@
...
+    :type is_folder: bool
     """
     items_list = []
 
     for item in items:
         # create a list item using the song filename for the label
         li = xbmcgui.ListItem(label=items[item]['title'])
         # set the fanart to the album cover
-        li.setProperty('fanart_image', os.path.join(ADDON_FOLDER, 'resources/media/fanart.jpg'))
+        li.setProperty(
```
Comment (enen92) [inline plugin.audio.eco99music/addon.py]: Please use `listitem.setArt`

Comment (noamsto) [reply plugin.audio.eco99music/addon.py]: I actually see that it's done at line 132-133

**Comment by Rechi** [comment]
@nire0510 please approve or deny this changes.
We only accept changes the add-on maintainer submits or approves.

**Comment by nire0510** [comment]
Approved, thanks!

**Comment by Rechi** [comment]
@noamsto this add-on isn't Python 3 compatible. Future updates have to be compatible with Python 3.
Have a look at https://kodi.tv/article/attention-addon-developers-migration-python-3 for more information.

**Comment by noamsto** [comment]
Will take that into account.

**Pull request merged**